### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Mwapsam**
### Blog app - Views
**In this project I added the Views associated with the Models and Controllers I created for the Blog App. To achieve this I did the following:**

_- Use preprocessed HTML file with embedded Ruby code._

_- Use layouts and templates for shared content._

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,17 @@
...
+            <li><%= link_to post.title, {:action => 'show', :id => post.id}%></li>
+            <p><%= post.text %></p>
+            <h3>Comment <%= post.comments.count%>,  <span>Likes <%= post.likes.count%></span></h3>
+        </div>
+    <div class="comments">
+        <% post.comments.each do |comment| %>
+            <div class="comment">
+                <p><%= comment.text %></p>
+            </div>
+        <% end %>
```
Comment (rukundoeric) [inline app/views/posts/index.html.erb]: - [ ] <b>REQUIRED:</b> The project requires to only render the 5 recent_comments here not all of them. Kindly do that to comply with project requirements.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Mwapsam, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (Mwapsam) [reply app/views/posts/index.html.erb]: Thanks for pointing it out. Done 😀

**Review by Lameck1 — APPROVED** [review]
Hi, @Mwapsam,

You've done well addressing the concern from the previous review 👏🏼.
Tasks for this project are now successfully implemented 💯.

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**